### PR TITLE
Deprecate imodel-select & markup-frontstage

### PR DESCRIPTION
**Pull request by aruniverse**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,23 @@ This repository contains a set of npm packages that deliver React utilities (Com
...
+
+The following packages have been deprecated and will no longer be maintained:
+| Name                               | Folder                                                         | Changelog | Version                                                                                                                                                |
+| ---------------------------------- | -------------------------------------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| @bentley/imodel-select-react       | [packages/imodel-select](./packages/imodel-select/)            | [changelog](./packages/imodel-select/CHANGELOG.md) | [![npm version](https://badge.fury.io/js/%40bentley%2Fimodel-select-react.svg)](https://badge.fury.io/js/%40bentley%2Fimodel-select-react)             |
```
Comment (calebmshafer) [inline README.md:24]: Do we want to point to the admin component that should be used instead of this one?

Comment (aruniverse) [reply README.md:24]: Yea i was wondering if i should do that here in the root readme or in the pkg readme, but then again i dont think with the way i have it setup it won't publish a new version of the pkg. 

I'll update to add a tab for replacements (pointing to admin-comp-react)

Comment (aruniverse) [reply README.md:24]: Do we actually want to publish a new ver saying this pkg has been deprecated?

**Review by calebmshafer — APPROVED** [review]
Let @skirby1996 know if you want us to run the `npm deprecate` script for it.

We'll just need the message you want to send along with it.

**Pull request merged**